Makefile

```diff
@@ -2,10 +2,16 @@
 sync:
 	poetry install --extras dev_all --sync
 
-update:
-	poetry update --extras dev_all
```

Comment:
I dunno if I added this and never used it or what, but as far as I can tell that option never worked in poetry.